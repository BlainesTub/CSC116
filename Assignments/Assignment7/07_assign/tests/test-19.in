log(exp(x+2.0)/exp(x+2.0) + 5)
2