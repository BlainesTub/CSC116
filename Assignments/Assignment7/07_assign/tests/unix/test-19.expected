f(x) = log(exp(x + 2)/exp(x + 2) + 5)
f(2) = 1.79176
f'(x) = ((exp(x + 2)*(1 + 0)*exp(x + 2) - exp(x + 2)*exp(x + 2)*(1 + 0))/exp(x + 2)^2 + 0)/(exp(x + 2)/exp(x + 2) + 5)
f'(2) = 0
